build-utils/search-untranslated.js

```diff
@@ -33,7 +33,7 @@ const recursive = require('recursive-readdir');
 
     const fileContent = fs.readFileSync(filePath).toString();
     let match;
-    while (match = searchExp.exec(fileContent)) {
+    while ((match = searchExp.exec(fileContent))) {
```

Comment:
Don't understand this change

Reply:
`Fixed warning: 'match = searchExp.exec(fileContent)' used as condition`

I believe that by adding the parens, we communicate we *want* to do an assignment first and then use the truthiness of that as our condition. Otherwise, the reader might think we forgot a `=` or two.

Reply:
That's kind of strange.  I'm not sure the extra patentheses actually communicate that.  Is this generally used convention?